docs/patterns/existing-eks-observability-accelerators/existing-eks-apiserver-observability.md

```diff
@@ -2,7 +2,7 @@
 
 ## Objective 
 
-This pattern aims to add Observability on top of an existing EKS cluster and adds API server monitoring, with open source managed AWS services.
+This pattern aims to add Observability and Control plane logging on top of an existing EKS cluster and adds API server monitoring, with open source managed AWS services.
```

Comment:
This change is not required. I would rather add a bullet to say. If Control Plane logging is not enabled.. please use enableControlPlane logging function attaching a small relevant code snippet.